### ISSUE TITLE
Variable Pricing products are not working with Custom Deliverables

### ISSUE THREAD

**Issue by mintplugins** [easydigitaldownloads/edd-custom-deliverables#20]
Variable products with custom deliverables attached are showing this error:

```
Invalid argument supplied for 
foreach() in /wp-content/plugins/edd-custom-deliverables/includes/functions/helper-functions.php 
on line 48 
```

**Comment by mintplugins** [comment]
Merged

**Comment by mintplugins** [comment]
This also fixes #18
